test/common/testlib.py

```diff
@@ -912,8 +912,13 @@ def tearDown(self):
             self.check_browser_errors()
         shutil.rmtree(self.tmpdir)
 
-    def login_and_go(self, path=None, user=None, host=None, authorized=True, superuser=True, urlroot=None, tls=False):
+    def login_and_go(self, path=None, user=None, host=None, authorized="unset", superuser=True, urlroot=None, tls=False):
```

Comment:
Annoying nitpick from the peanut gallery: The canonical value for "unset" is `None` in Python.

Reply:
Yeah, but `login_and_go(authorized=None, superuser=...)` needs to mean something different than `login_and_go(superuser=...)`... This sure is ugly; I try to come up with something better.